doc/tutorial.rst

```diff
@@ -586,8 +573,9 @@ The `static/js/src/simplevideo_edit.js` JavaScript file should look like this: :
           maxwidth: $(element).find('input[name=maxwidth]').val(),
           maxheight: $(element).find('input[name=maxheight]').val()
         };
+        runtime.notify('save', {state: 'start'});
         $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
-          window.location.reload(false);
+          runtime.notify('save', {state: 'end'});
```

Comment:
We don't use the xblock initialization script in our tutorial, so perhaps we should just remove it from here. We're not going to have to worry about someone ending up with the vertical block if they're following this. It would also remove the confusion on what it's for, as its presence implies it's needed to have multiple blocks on the same page, and so far as I know, it shouldn't be.